CONTRIBUTING.md

```diff
@@ -0,0 +1,103 @@
+# So You Want to Contribute to Papermill...
+We welcome all contributions to Papermill both large and small. We encourage you
+to join our community.
+
+## Our Community Values
+
+We are an open and friendly community. Everybody is welcome.
+
+We encourage friendly discussions and respect for all. There are no exceptions.
+
+All contributions are equally important. Documentation, answering questions, and
+fixing bugs are equally as valuable as adding new features.
+
+Check out <https://www.python.org/psf/codeofconduct/> for the Python code of conduct.
```

Comment:
nteract has its own code of conduct, so we should probably link to that: https://github.com/nteract/nteract/blob/master/CODE_OF_CONDUCT.md